src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
```

Comment:
Every invoke of method create new object of UserService, you can save it in field of class